docs/events/timeline-ui-overview.asciidoc

```diff
@@ -2,84 +2,72 @@
 = Investigate events in Timeline
 
 Use Timeline as your workspace for investigations and threat hunting.
-Data from multiple indices can be added to a Timeline, which enables
-investigating complex threats.
+You can add data from multiple indices to a Timeline to facilitate advanced investigations.
 
-You can drag or send items of interest to Timeline to create the query you need
-to investigate an alert or event. You can add items from tables and histograms
+You can drag or send items of interest to a Timeline to create the desired query. You can add items from tables and histograms
 on the *Overview*, *Alerts*, *Hosts*, and *Network* pages, as well as from
-within a Timeline itself. Additionally, you can add a query directly in Timeline
-(click *+ Add field*).
+other Timelines. Additionally, you can add a query directly in Timeline
+by clicking *+ Add field*.
 
 [role="screenshot"]
 image::images/timeline-ui-updated.png[]
```

Comment:
This image is outdated. Here's the current one:


<img width="1847" alt="timeline-ui-updated" src="https://user-images.githubusercontent.com/79124755/156790445-76c8f6e2-6259-40dd-a516-fee2d726fe25.png">